ansible_runner/cleanup.py

```diff
@@ -79,9 +79,9 @@ def is_alive(dir):
 
     try:
         os.kill(pid, signal.SIG_DFL)
```

Comment:
While we are working on linting, could we please please rework this? in this line the feature of glibc is used that signal 0 checks if the caller has permissions to send a signal to the PID. for some reason somebody passed the default signal handler which evaluates `0` in there.

so `os.kill(pid, 0)` would be more correct but the problem still remains that this line uses a feature that is NOT part of the `os.kill` contract and it checks for permission of the pid, not existence. (so if the process does exist but runner is not allowed to send a signal which can happen for multiple reasons, it just assumes it is not there)

Reply:
Let's not try to fix random things within this PR. The linting change will likely have to be backported to get our CI working correctly across all branches. I'm not convinced a change to this line should be backported yet.

Reply:
fair!